components/bokeh/bokeh.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import * as Bokeh from '@bokeh/bokehjs'
+
+
+function randid() {
+    const S4 = function () {
+        return (((1 + Math.random()) * 0x10000) | 0).toString(16).substring(1);
+    };
+    return (S4() + S4() + "-" + S4() + "-" + S4() + "-" + S4() + "-" + S4() + S4() + S4());
+}
+
+export default function BokehPlot({plot}) {
+    const ref = React.useRef(null)
+
+    React.useEffect(() => {
+        if (!ref.current || !plot) return
+        const id = randid()
+        const div = document.createElement('div')
+        div.id = id
+        ref.current.appendChild(div)
+        Bokeh.embed.embed_item(plot, id)
+        return () => {
+            ref.current
```

Comment:
This exit part of the hook should be removing the previously created element.

`ref.current.removeChild(div)`